### PR TITLE
awk: refresh/add pages

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

**Comment by EmilySeville7cfg** [comment]
@CleanMachine1, CI fails again.

**Comment by CleanMachine1** [comment]
It will fail with all PRs until.

File: pages/common/awk.md [hunk 1]
```diff
@@ -1,36 +1,33 @@
 # awk
 
 > A versatile programming language for working on files.
-> More information: <https://github.com/onetrueawk/awk>.
+> See also: `sed`, `ed`.
+> More information: <https://manned.org/awk.1>.
```
Comment (CleanMachine1) [inline pages/common/awk.md:5]:
```suggestion
> More information: <https://github.com/onetrueawk/awk>.
```

File: pages/common/awk.md [hunk 2]
```diff
@@ -1,36 +1,21 @@
...
+`{{command}} | awk '{ print gensub("apple", "mango", "g") }'`
 
-- Print the second column of the lines containing "foo" in a space-separated file:
+- Execute a specific script [f]ile and print the result to `stdout`:
 
-`awk '/{{foo}}/ {print $2}' {{filename}}`
+`{{command}} | awk -f {{path/to/script.awk}}`
 
-- Print the last column of each line in a file, using a comma (instead of space) as a field separator:
+- Print first fields of all input lines delimited by a specific delimiter (basic regex) to `stdout`:
```
Comment (kbdharun) [inline pages/common/awk.md:15]:
```suggestion
- Print the first fields of all input lines delimited by a specific delimiter (basic regex) to `stdout`:
```

File: pages/common/awk.md [hunk 3]
```diff
@@ -1,36 +1,21 @@
...
 
-`awk -F ',' '{print $NF}' {{filename}}`
+`{{command}} | awk -F ',' '{ print $1 }'`
 
-- Sum the values in the first column of a file and print the total:
+- Print first 10 fields of all input lines delimited by a single space (basic regex) to `stdout`:
```
Comment (kbdharun) [inline pages/common/awk.md:19]:
```suggestion
- Print the first 10 fields of all input lines delimited by a single space (basic regex) to `stdout`:
```

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution. I have some minor suggestions.

**Review by blueskyson — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
I am closing this PR as it's too outdated. Changes from it will be applied in separate PRs later.

**Pull request closed without merging**